mkdocs.yml

```diff
@@ -7,6 +7,7 @@ nav:
   - Static Sites: static-sites.md
   - API Reference:
     - api/index.md
+    - CLI: api/applications.md    
```

Comment:
This should be Applications: api/applications.md so that the page title and side bar are correct

Reply:
Fixed!